language/tools/move-cli/src/main.rs

```diff
@@ -625,6 +642,12 @@ fn main() -> Result<()> {
                 fs::remove_dir_all(&storage_dir)?;
             }
 
+            // delete packages
+            let package_dir = Path::new(&move_args.package_dir);
```

Comment:
I'll leave this comment elsewhere too, but we might want to put the package stuff either in storage. 

I like the feeling of it sitting in storage as like "here is this magical elsewhere code that we are preloading for you"

Reply:
@shb thoughts?

Reply:
+1 to keeping these package dirs around for the same reason. I also think in the eventual setting where you can have user-defined modes, having them be actual directories that you can see will be nice to have.

Reply:
After sleeping on it, I think for now we could just have a single hardcoded directory, that sits outside of build or storage. We can deal with custom packages later

Reply:
Did I miss some conversation that happened about this? I was coming to leave a comment that it should probably live in `build`. It is required to be a file on disk because of the compiler API it seems, which is a build-time thing, so that seems the appropriate place to put it. We should avoid creating too many top level directories in the users code to avoid name clashes and ugliness in `ls` output etc.

Reply:
Perhaps living in build isn't too bad.. but originally that had intended to be the directory for the output from `move build`

Reply:
I was thinking of it like Cargo's `target` where lots of things the "build" process deals with are stored there, not just binary artifacts, but also timestamp data, generated code, etc.

Reply:
Generalizing `build` to a `target`-like directory for generated stuff seems like a good idea to me.

Reply:
In this PR, we did re-use the `build` dir and produces the package data under `build/package/<package-name>/[source_files | compiled]` to reduce the number of top-level directories, as Jack and Todd suggested.